app/controllers/queue_items_controller.rb

```diff
@@ -33,4 +37,13 @@ def video_in_queue?(video)
   def queue_video(video)
     QueueItem.create(user: current_user, video: video, list_position: current_user.queue_items.count + 1) unless video_in_queue?(video)
   end
+
+  def update_queue_items
+    ActiveRecord::Base.transaction do
+      params[:queue_items].each do |queue_item_data|
+        queue_item = QueueItem.find(queue_item_data["id"])
+        queue_item.update_attributes!(list_position: queue_item_data["list_position"], rating: queue_item_data["rating"]) if queue_item.user == current_user
```

Comment:
Cool, I used update originally .. but changed it following the video. Should've kept it